src/asm/rgbasm.5

```diff
@@ -1053,10 +1074,9 @@ ANGLE SET ANGLE+256.0
 .Ed
 .Pp
 .Ss String Expressions
-The most basic string expression is any number of characters contained in double
-quotes ("for instance").
-Like in C, the escape character is \[rs], and there are a number of commands you
-can use within a string:
+The most basic string expression is any number of characters contained in double quotes
+.Pq \&"for instance" .
```

Comment:
Or `.Pq Qq for instance .`

Reply:
Considering this is a literal, source code string, I'm not comfortable with using a macro that might (and appeared to in the mdoc documentation's own render) change the quotes.

Reply:
Good catch! From the macro description I’m pretty sure that’s a bug. In terminal output it specifically selects straight quotes, not curly quotes. I’ll see if I can get that fixed in mandoc.

Reply:
I replaced it with `.Pq Ql \&"for instance" .`

Reply:
https://github.com/openbsd/src/commit/ac01d98ff55bc59c9c832169d23a7013a1e84516